### PR TITLE
Load theming.css in plonebrowser using z3c.jbot and more.

### PR DESCRIPTION
**Pull request by maethu**
This is imho the better solution, instead of using the js integration. 

Also some misc. fixes.
- Internal pathbar stylings:
  screen shot 2015-08-19 at 16 41 17
- Translations and Icons
  screen shot 2015-08-19 at 17 20 49

### CODE REVIEW

**Comment by maethu** [comment]
@jone ready for review!!

File: plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+  <title i18n:domain="plone.tinymce" i18n:translate="">Insert/Edit</title>
+  <script type="text/javascript" src="../../tiny_mce_popup.js"></script>
+  <script type="text/javascript" src="../../utils/mctabs.js"></script>
+  <script type="text/javascript" src="../../utils/form_utils.js"></script>
+  <script type="text/javascript" src="js/plonebrowser.js"></script>
+  <link rel="stylesheet" type="text/css" href="css/plonebrowser.css" />
+  <link rel="stylesheet" type="text/css" tal:attributes="href string:${portal_url}/portal_css/columns.css" />
+  <link rel="stylesheet" type="text/css" href="css/plonebrowser.css" />
+  <!-- PATCH START -->
+  <link rel="stylesheet" type="text/css" tal:attributes="href string:${portal_url}/theming.css" />
```
Comment (jone) [inline plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt]: This kind breaks caching. We probably should generate the URL with the helper function (in `ftw.theming.browser.theming_css`), that means we must make it traversable.. :confused:

Comment (maethu) [inline plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt]: Are you going to add this feature in ftw.theming, or should I tomorrow?

Comment (jone) [inline plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt]: I'm doing it. https://github.com/4teamwork/ftw.theming/issues/14

Comment (jone) [inline plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt]: https://github.com/4teamwork/ftw.theming/issues/14

Comment (jone) [inline plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt]: @maethu Done in https://github.com/4teamwork/ftw.theming/pull/15

**Comment by maethu** [comment]
@jone theme's to work fine: 

screen shot 2015-08-20 at 10 21 49

It's also fast again :sparkles:

**Pull request merged**